server/src/com/thoughtworks/go/config/update/RoleConfigDeleteCommand.java

```diff
@@ -38,11 +38,16 @@ public void update(CruiseConfig preprocessedConfig) throws Exception {
         }
 
         removeFromServerRole(preprocessedConfig, preprocessedRole);
+        removeFromAdmins(preprocessedConfig);
```

Comment:
Does this mean we might have bad configuration that needs cleanup (via XSL transform)?